app/scripts/fetch_xulrunner

```diff
@@ -483,6 +485,14 @@ if [ $BUILD_LINUX == 1 ]; then
 		if [ -e "$archived_file" ]; then
 			echo "Using $archived_file"
 			cp "$archived_file" "firefox-$GECKO_VERSION.tar.bz2"
+		elif [[ "$arch" == "aarch64" ]]; then
+			curl -O "http://ftp.de.debian.org/debian/pool/main/f/firefox-esr/firefox-esr_102.15.0esr-1~deb11u1_arm64.deb"
+			ar xOf firefox-esr_102.15.0esr-1\~deb11u1_arm64.deb data.tar.xz
```

Comment:
Why `ar`? This needs to run on macOS as well, which has a different version of `ar`. Just using `tar` for this as well seems to work fine?